LocalReferenceLinkRel/explainer.md

```diff
@@ -148,6 +148,79 @@ the shadow root where they are defined, as illustrated by the following examples
   Styles defined inside the sibling Shadow Root are not applied, so "Inside Sibling Shadow DOM" is not blue.
 </p>
 
+### Fetch Behavior
+
+The current behavior for the `<link>` tag with following markup in a file
+named "foo.html" is as follows:
+
+```html
+<style id="style_tag">
+  p {
+    color: blue;
+  }
+</style>
+<template shadowrootmode="open">
+  <link rel="stylesheet" href="#style_tag" />
+  <p>Inside Shadow DOM</p>
+</template>
+```
+
+1. A fetch is initiated for `foo.html#style_tag` (a `<base>` tag may modify
+the base URL).
+2. Upon resolving the fetch (this will usually be a cache hit for the current
+page), the `<link>` tag's `onerror` event is fired due to a MIME type mismatch
+(the `<link>` tag expects a CSS MIME type when `rel="stylesheet"`, while
+`foo.html#style_tag` is an HTML MIME type). Note that some
+User Agent's don't follow this behavior this and instead fire `onload`.
+
+There are several options to avoid this fetch:
+1. Using a different value for `rel` than `stylesheet` (`inline-stylesheet`
+is one option).
+2. Using a different attribute than `href` for in-document stylesheets.
+3. Using a special URL scheme, e.g. inline:id rather than relying on fragment
+identifiers.
+
+However, this may not be necessary. An easy method of polyfilling this behavior
+is to simply add `onerror` and `onload` handlers that look up the `<style>`
+element referenced and copy its contents into a dataURI, as follows:
+
+```html
+<style id="style_tag">
+    span {color: blue;}
```

Comment:
This is really picky, but since this example is presented as a parallel to the one above without the polyfill, it'd be good to have them match wherever possible: the style rule can be for `p` instead of span, the element inside the shadow can be `<p>Inside Shadow DOM</p>` instead of `<span>Shadow DOM</span>`, and the `<span>Light DOM text</span>` element can be removed.